packages/components/ng-ovh-responsive-popover/src/class/directive.js

```diff
@@ -1,58 +0,0 @@
-/**
- *  @ngdoc directive
- *
- *  @name responsivePopover.directive:responsivePopoverClass
- *
- *  @description
- *  This directive manage the way the popover is displayed. This uses the configuration setted into
- *  the `responsivePopoverProvider` to detect if the popover needs to be displayed on full screen
- *  or to be displayed normally.
- *  It is automatically setted by `responsivePopover` directive.
- *
- *  **Note:** when ui-bootstrap version >= 2.0 will be used, the only thing to do should be to add
- *  a custom class with the 'popover-class' option of the uibPopover directive. To be tested.
- */
-export default /* @ngInject */ function(matchmedia, responsivePopover) {
-  return {
-    restrict: 'C',
-    link(scope, iElement) {
-      const mobileMediaQuery = responsivePopover.getMobileMediaQuery();
-      let isMobileDetected = false;
-      let mediaQueryHandler = false;
-      let scrollPosition;
-      const resetFixedBody = function(scrollTo) {
-        $('body')
-          .removeClass('responsive-popover-fixed-body')
-          .css('top', '')
-          .scrollTop(scrollTo);
-      };
-
-      // on method returns a function that remove the media query listener
-      // @see https://github.com/AnalogJ/matchmedia-ng/blob/master/matchmedia-ng.js#L75
-      mediaQueryHandler = matchmedia.on(mobileMediaQuery, (mediaQueryList) => {
-        isMobileDetected = mediaQueryList.matches;
-
-        if (isMobileDetected) {
-          scrollPosition = $('body').scrollTop();
-          iElement.addClass('popover-full-screen');
```

Comment:
Can find this className into following file:
- packages/components/ng-ovh-responsive-popover/src/index.less

Also able to find the same className across multiple location:
- packages/components/ng-ovh-actions-menu/src/index.less
- packages/manager/apps/telecom/src/app/common/less/mixins.less
- packages/manager/apps/telecom/src/components/telecom/telephony/group/number/feature/conference/conference.less
- packages/manager/apps/telecom/src/components/telecom/telephony/group/number/number.less
- packages/manager/apps/telecom/src/components/telecom/telephony/scheduler/scheduler.less
- packages/manager/apps/telecom/src/components/telecom/telephony/service/choice-popover/choice-popover.less
- packages/manager/apps/telecom/src/components/telecom/telephony/timeCondition/time-condition.less

Don't know if we are relying on the same CSS rules.

Reply:
The directive in itself, isn't integrated in main angular module of the component: 

- It is define, and declared in its own module: https://github.com/ovh/manager/blob/master/packages/components/ng-ovh-responsive-popover/src/class/ovh-angular-responsive-popover-class.js, which is then exported.
.- But no import of this exported module: https://github.com/ovh/manager/blob/master/packages/components/ng-ovh-responsive-popover/src/index.js

However the less file can be clean to remove the unexisting class

Reply:
In fact I did delete this directive regardless of these usecases, as the module doesn't include it. But in fact there's a bug in manager telecom due to the absence of this directive. I'll change the code and try to replace jQuery by vanilla js.

Thx for the point